.github/workflows/bench.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           nix_path: nixpkgs=channel:nixos-unstable
           extra_nix_config: |
             trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
-            substituters = https://hydra.iohk.io https://cache.nixos.org/
+            substituters = https://cache.iog.io https://cache.nixos.org/
```

Comment:
This change could be in a different commit, but it doesn't matter too much, maybe not worth the hassle

Reply:
Since I had to change the commit anyways, I moved this change into a separate commit. :+1: